CHANGELOG.md

```diff
@@ -8,3 +8,28 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 [Unreleased]: https://github.com/puppetlabs/prm/compare/9d7ff6acd89f19cd24492acc9058c4c5752bcbe5..main
+
+- [(GH-263)](https://github.com/puppetlabs/pdkgo/issues/263) Added initial implementation based on pdkgo project structure.
+- [(GH-264)](https://github.com/puppetlabs/pdkgo/issues/264) Scaffold PRMlib
+- [(GH-265)](https://github.com/puppetlabs/pdkgo/issues/265) Scaffold tool & related structs
+- [(GH-266)](https://github.com/puppetlabs/pdkgo/issues/266) Scaffold backend interface
+- [(GH-267)](https://github.com/puppetlabs/pdkgo/issues/267) Scaffold PRM Config
+- [(GH-268)](https://github.com/puppetlabs/pdkgo/issues/268) Scaffold Docker backend
```

Comment:
IMO, these are maintainer facing PRs as they do not yet expose any functionality; a changelog is end-user facing documentation for what has changed, so I don't think we need these entries.